### PR TITLE
Add the ability to reload template sensors

### PR DESCRIPTION
**Pull request by mariwing**
Add support for reloading Template Sensors.

The service template.reload was added in home-assistant/core#39075

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -237,6 +237,11 @@
         "title": "Reload KNX",
         "category": "Home Assistant"
       },
+      {
+        "command": "vscode-home-assistant.templateReload",
+        "title": "Reload Template Sensors",
```
Comment (frenck) [inline package.json:242]: Small suggestion, as this reload isn't limited to Sensor, but covers most other entity platforms supported by the Template integration as well.

```suggestion
        "title": "Reload Template Entities",
```

**Review by frenck — CHANGES_REQUESTED** [review]

**Comment by frenck** [comment]
Tested locally, works fine 👍  Just the small language suggestion.

File: package.json (hunk 1 above)
Comment (mariwing) [reply package.json:242]: Thanks for the quick review and good feedback, I wasn't really aware of that so then I learned something today 👍

**Review by frenck — APPROVED** [review]
Awesomeness! 👍

**Pull request merged**